less/components/consonant/filters/top/item.less

```diff
@@ -55,8 +55,7 @@
         &-name {
             display: flex;
             align-items: center;
-
-            .no-margin;
+            margin: 2 0;
```

Comment:
It should be: `margin: 2px 0;`

Reply:
You should also add a _Resolves: <Link to jira ticket>_ section for easy cross reference :)

Reply:
Both things done @cmiqueo :)